gtsam/sfm/SfmTrack.h

```diff
@@ -35,28 +35,26 @@ typedef std::pair<size_t, Point2> SfmMeasurement;
 typedef std::pair<size_t, size_t> SiftIndex;
 
 /**
- * @brief An SfmTrack stores SfM measurements grouped in a track
- * @addtogroup sfm
+ * @brief Track containing 2D measurements associated with a single 3D point.
+ * Note: Equivalent to gtsam.SfmTrack, but without the 3d measurement.
+ * This class holds data temporarily before 3D point is initialized.
  */
-struct GTSAM_EXPORT SfmTrack {
-  Point3 p;       ///< 3D position of the point
-  float r, g, b;  ///< RGB color of the 3D point
-
+struct GTSAM_EXPORT SfmTrack2d {
   /// The 2D image projections (id,(u,v))
   std::vector<SfmMeasurement> measurements;
 
-  /// The feature descriptors
+  /// The feature descriptors (optional)
```

Comment:
just curious, what is a siftIndex? I see it marked as a descriptor, so i would expect something like a 128-d vector, but that doesnt match the type (pair of size_t's). So it seems like it is supposed to represent 2d detections? Or maybe I'm misunderstanding the data structure?

Reply:
Don't remember and not on my A->B at this time.